File: 프로그래머스/1/135808. 과일 장수/과일 장수.js
function solution(k, m, score) {
    
    const sort = score.sort((a, b) => b - a);
    
    let prices = 0;
    
    for(let i = 1; i <= Math.floor(score.length / m); i++){
        
        prices += sort[(i) * m - 1] * m
    }
    
    return prices
}

File: 프로그래머스/1/135808. 과일 장수/과일 장수.py
def solution(k, m, score):
    order = sorted(score, reverse=True)
    
    numOfBox = len(score) // m
    
    price = 0
    
    for i in range(numOfBox):
        
        price += (order[(i + 1) * m - 1] * m)
        
    return price